ui/src/app/subnets/views/SubnetDetails/SubnetDetailsHeader/SubnetDetailsHeader.test.tsx

```diff
@@ -31,3 +34,18 @@ it("does not show a spinner subtitle if the subnet is detailed", () => {
 
   expect(screen.queryByTestId("section-header-subtitle-spinner")).toBeNull();
 });
+
+it("displays available actions", () => {
+  const subnet = subnetDetailsFactory({ id: 1, name: "subnet-1" });
+  render(<SubnetDetailsHeader subnet={subnet} />);
+
+  Object.values(subnetActionLabels).map((name) => {
```

Comment:
Looks like these should be using `forEach()` (or some other loop that doesn't expect a return value) instead of `map()`

Reply:
Ah yeah good point 👍